doc/sphinx/source/data/new-commondata.rst

```diff
@@ -0,0 +1,205 @@
+
+    version: 1
+    version_comment: "Initial implementation"
+
+    # References
+    arXiv:
+        url: ""
+    iNSPIRE:
+        url: "https://inspirehep.net/literature/302822"
+    hepdata:
+        url: "https://www.hepdata.net/record/ins302822"
+        version: 1
+
+    nnpdf_metadata:
+        nnpdf31_process: "PROCESS"
+        experiment: "EXPERIMENT_NAME"
+
+    implemented_observables:
+    - observable_name: "OBS"
+        observable:
+            description: "Description of the observable"
+            label: "Latex label for the observable"
+            units: "[u]"
+        ndata: n_of_datapoints
+        tables: (optional) [n, j, k] # corresponding tables in the hepdata entry
+        npoints: (optional) [n, j, k] # number of points per table
+
+        # Plotting information
+        dataset_label: "Label to be used in reports"
+        plot_x: variable_1
+        figure_by:
+            - variable_2
+        kinematic_coverage:
+            variable_1: var_1
+            variable_2: var_2
+
+        kinematics:
+            variables:
+                var_1: {description: "Description of var", label: latex, units: "u"}
+                var_2: {description: "Description of var", label: latex, units: "u"}
+                var_3: {description: "Description of var", label: latex, units: "u"}
+            file: kinematics.yaml
+
+        data_central: data.yaml
+        data_uncertainties:
+            - uncertainties.yaml
+            - uncertainties_2.yaml
+
+        # Having varians is optional
+        # variants can overwrite the data_uncertainties 
+        variants:
+            different_errors:
+                data_uncertainties:
+                    - uncertainties.yaml
+                    - uncertainties_3.yaml
+
+        # The theory field is always optional
+        theory: 
+            FK_tables:
+                - - DYE605
+            operation: 'null'
+
+
+Versioning
+~~~~~~~~~~
+
+The initial version of a dataset should be set to ``version: 1``.
+Any change on a dataset should be *always* accompanied of a version bump and a ``version_comment`` explaining the update.
+This will allow to keep an exact tracking of all changes to every dataset even if they change over time.
+
+Variants
+~~~~~~~~
+
+In some ocassions we might want to maintain two variations of the same observable.
+For instance, we might have two incompatible sources of uncertainties. In such case a variant can be added.
+The syntax of the ``variants`` is:
+
+Theory
+~~~~~~
+
+The theory field defines how predictions for the dataset are to be computed.
+It includes two entries:
+
+- ``FK_tables``: this is a list of lists which defines the fktables to be loaded. The outermost list are the operands (in case an operation is needed to recover the observable, more on that below). The innermost list are the grids that are to be concatenated in order to form the operands.
+- ``operaton``: operation to be applied in order to compute the observable
```

Comment:
we definitely need some cross link here ... I don't know if and where the operations are explained, though